packages/flutter/lib/src/material/text_field.dart

```diff
@@ -243,6 +249,7 @@ class _TextFieldState extends State<TextField> {
         selectionControls: materialTextSelectionControls,
         onChanged: widget.onChanged,
         onSubmitted: widget.onSubmitted,
+        onSelectionChanged: (TextSelection _, bool longPress) => _onSelectionChanged(context, longPress),
```

Comment:
interesting, i would have expected this to belong to the selection controls delegate. This works too.